driver/ppm_fillers.c

```diff
@@ -1342,23 +1389,44 @@ int f_proc_startupdate(struct event_filler_arguments *args)
 				{
 					struct super_block *sb = NULL;
 					unsigned long sb_magic = 0;
-
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 18, 0)
+					sb = exe_file->f_path.dentry->d_sb;
+#else
 					sb = exe_file->f_inode->i_sb;
+#endif
 					if(sb)
 					{
 						sb_magic = sb->s_magic;
 						if(sb_magic == PPM_OVERLAYFS_SUPER_MAGIC)
 						{
+							struct ovl_entry *oe = (struct ovl_entry*)(exe_file->f_path.dentry->d_fsdata);
```

Comment:
Since this code is basically called twice, can we extract a function?
Same in bpf driver.